runtime/opt/taupage/init.d/01-register-scalyr-agent.sh

```diff
@@ -201,9 +201,29 @@ then
   fi
 fi
 
+echo -n "adding max_line_size... ";
+sed -i '/api_key/a \  max_line_size: 49900,' $scalyr_config
+if [ $? -eq 0 ];
+then
+    echo "DONE"
+else
+    echo "ERROR"
+    exit
```

Comment:
you should always exit with !=0 in case of error, so the automated checks during taupage build will notice if it failed

Reply:
Good idea!